src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
```

Comment:
1. Naming: The class and method names are appropriate and descriptive. Good job on that!